src/core/qgsvectorlayereditutils.h

```diff
@@ -189,6 +189,12 @@ class CORE_EXPORT QgsVectorLayerEditUtils
      */
     bool boundingBoxFromPointList( const QList<QgsPointXY> &list, double &xmin, double &ymin, double &xmax, double &ymax ) const;
 
+    /**
+     * Return default Z value
+     * Use for set Z coordinate to edited vertex for 2.5d geometries
+     */
```

Comment:
Can you add `\since QGIS 3.0` and take a look on typo (like `to edited` -> `to edit`)?